src/main/java/org/apache/commons/lang3/builder/ToStringStyle.java

```diff
@@ -2608,7 +2608,7 @@ private boolean isJsonObject(final String valueAsString) {
          * @param value the value to append.
          */
         private void appendValueAsString(final StringBuffer buffer, final String value) {
-            buffer.append('"').append(StringEscapeUtils.escapeJson(value)).append('"');
+            buffer.append('"').append(value).append('"');
```

Comment:
Directly writing characters causes JSON special characters to not be properly escaped.

Reply:
so how to solve this problem.